coordinator/src/admin.rs

```diff
@@ -138,7 +138,7 @@ pub async fn list_dlc_channels(
 pub async fn list_on_chain_transactions(
     State(state): State<Arc<AppState>>,
 ) -> Result<Json<Vec<TransactionDetails>>, AppError> {
-    let transactions = state.node.inner.get_on_chain_history().await.map_err(|e| {
+    let transactions = state.node.inner.get_on_chain_history().map_err(|e| {
```

Comment:
🔧 We should wrap this in `spawn_blocking`/`block_in_place`.

Reply:
Are you suggesting to do both: `block_in_place` and then `spawn_blocking`?

Reply:
Addressed here - https://github.com/get10101/10101/pull/950/commits/8fc8096e9a69a7415c9afe4e0b41dd0fabffc7b1

Please have a look if this is what you intended.